frontend/src/main/utils/useBackend.jsx

```diff
@@ -2,29 +2,6 @@ import { useQuery, useMutation, useQueryClient } from "react-query";
 import axios from "axios";
 import { toast } from "react-toastify";
 
-// example
```

Comment:
Why were these comments removed?  This is not commented out code, but rather it is documentation for this module.  

Can we restore this?